code/obj/window.dm

```diff
@@ -987,17 +987,27 @@
 
 
 	auto
-		name = "reinforced autowindow grille spawner"
+		name = "autowindow grille spawner (will apply in time)"
```

Comment:
what is this even supposed to mean

Reply:
Once I convert maps to use the new paths of auto/reinforced, I'll submit a PR to make auto place normal windows. At the moment however, both paths still place reinforced windows.

Reply:
i am bad explainer forgive me. basically i plan to make normal wingrilles. the /auto path and /auto/reinforced both place reinforced windows right now. but, when i convert maps to /auto/reinforced after this pr is merged, i will make pr that make /auto place nonreinforced windows.

Reply:
thats why it says (will apply in time), the path will place nonreinforced when i make the pr.